src/pages/Integrations/SplunkSetup/Constants.ts

```diff
@@ -0,0 +1,23 @@
+export const RBAC_GROUPS_API = '/api/rbac/v1/groups/';
+export const RBAC_ROLES_API = '/api/rbac/v1/groups/{}/roles/';
+export const RBAC_ADD_USER_API = '/api/rbac/v1/groups/{}/principals/';
+export const CREATE_INTEGRATION_API = '/api/integrations/v1.0/endpoints';
+export const CREATE_BEHAVIOR_GROUP_API = '/api/notifications/v1.0/notifications/behaviorGroups';
+export const UPDATE_BEHAVIOR_GROUP_API = '/api/notifications/v1.0/notifications/behaviorGroups/{}/actions';
+export const EVENTTYPE_API = '/api/notifications/v1.0/notifications/eventTypes/{}/behaviorGroups';
+
+// If prod use prod UUID, else use stage
+export const NOTIF_ADM_ROLE_UUID = window.insights?.chrome.getEnvironment() === 'prod' ? '' : '15f910ad-150a-46dd-b666-0b21088e9c55';
+export const BEHAVIOR_GROUP_BUNDLE_ID =  window.insights?.chrome.getEnvironment() === 'prod' ? '' : '35fd787b-a345-4fe8-a135-7773de15905e';
```

Comment:
Needs to be replaced with API calls - this is a specific bundle and admin role?

Reply:
Yes, Notifications Administrator and the bundle is RHEL